WcaOnRails/lib/time_limit.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+class TimeLimit
+  TEN_MINUTES_IN_CENTISECONDS = 10*60*100
+
+  attr_accessor :centiseconds, :cumulative
```

Comment:
I see!

Yeah, I don't know about things at that level. See point 4 in the numbered list...